packages/middleware-endpoint/package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "@aws-sdk/middleware-endpoint",
+  "version": "3.130.0",
+  "scripts": {
+    "build": "concurrently 'yarn:build:cjs' 'yarn:build:es' 'yarn:build:types'",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
+    "build:types": "tsc -p tsconfig.types.json",
+    "build:types:downlevel": "downlevel-dts dist-types dist-types/ts3.4",
+    "clean": "rimraf ./dist-* && rimraf *.tsbuildinfo",
+    "test": "jest --passWithNoTests"
+  },
+  "main": "./dist-cjs/index.js",
+  "module": "./dist-es/index.js",
+  "types": "./dist-types/index.d.ts",
+  "author": {
+    "name": "AWS SDK for JavaScript Team",
+    "url": "https://aws.amazon.com/javascript/"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@aws-sdk/signature-v4": "*",
+    "@aws-sdk/types": "*",
+    "@aws-sdk/util-config-provider": "*",
+    "@aws-sdk/util-middleware": "*",
+    "tslib": "^2.3.1"
+  },
+  "devDependencies": {
+    "@aws-sdk/node-config-provider": "*",
+    "@tsconfig/recommended": "1.0.1",
+    "concurrently": "7.0.0",
+    "downlevel-dts": "0.7.0",
```

We bumped to 0.10.1 in https://github.com/aws/aws-sdk-js-v3/pull/3938
```suggestion
    "downlevel-dts": "0.10.1",
```